common/djangoapps/student/views/dashboard.py

```diff
@@ -803,7 +804,17 @@ def student_dashboard(request):
         if enrollment.course_overview.pre_requisite_courses
     )
     courses_requirements_not_met = get_pre_requisite_courses_not_completed(user, courses_having_prerequisites)
-
+    sku_not_enrollment_in_requirement = None
+    run_extension_point(
+        'PEARSON_CORE_SORT_ENROLLED_PREREQUISITES',
+        user=user,
+        courses_requirements_not_met=courses_requirements_not_met,
+    )
+    sku_not_enrollment_in_requirement = run_extension_point(
```

Comment:
What happens in case there are multiple courses with prerequisites? @anfbermudezme

Reply:
To fix this behavior I made changes to the three PR'S, This and 

[](https://github.com/Pearson-Advance/openedx-themes/pull/189 )
[](https://github.com/Pearson-Advance/pearson-core/pull/25)

thanks !